project/scripts/release.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Usage:
+# RELEASE_DIR=<directory to place fresh clones of projects> ./release.sh
+
+set -e
+
+# TODO: clone all the projects
+    # cd $RELEASE_DIR
+    # git@github.com:lampepfl/dotty
+    # git clone git@github.com:lampepfl/dotty-example-project.git
+    # git clone --single-branch --branch mill git@github.com:lampepfl/dotty-example-project.git
+    # git clone git@github.com:lampepfl/dotty.g8.git
+    # git clone git@github.com:lampepfl/dotty-cross.g8.git
+    # git clone git@github.com:lampepfl/homebrew-brew.git
+    # git clone git@github.com:lampepfl/packtest.git
+    # git clone git@github.com:scalacenter/scastie/.git
+
+# Parsing latest release from the list of tags not ending in RCX
+version=($(git tag --list '*.0' --sort=v:refname | tail -n1 | tr '.' '\n'))
+
+major=${version[0]}
+minor=${version[1]}
+build=${version[2]}
+
+LAST_RELEASE="$major.$minor"
+
+RELEASE="$major.$((minor+1))"
+
+RC_RELEASE="0.$((minor+2))"
```

Comment:
We've been releasing `0.x` stable and `0.(x + 1)-RC1` at the same time but there is no reason it should always be the case